packages/core/src/fhirpath/fhirpath.test.ts

```diff
@@ -2976,103 +2976,81 @@ describe('FHIRPath Test Suite', () => {
 
   describe('testBooleanLogicOr', () => {
     test('testBooleanLogicOr1', () => {
-      expect(evalFhirPath('(true or true) = true', patient)).toEqual([true]);
+      expect(evalFhirPath('(true or true)', patient)).toEqual([true]);
```

Comment:
All of these test strings were originally ported from a FHIRPath test suite.  My preference would be to leave them as-is unless there is good reason.

For example, this line originates from here: https://github.com/HL7/FHIRPath/blob/master/tests/r4/tests-fhir-r4.xml#L972

Reply:
Ah, sounds good, I'll change it back; I just found it easier to debug this way